org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -207,3 +207,37 @@ GithubOidcSageBionetworksScicompProvisioner:
   DefaultOrganizationBinding:
     Account: !Ref ScicompAccount
     Region: us-east-1
+
+GithubOidcSageBionetworksServiceCatalogLibrary:
```

Comment:
hmm, this is exactly the same as `GithubOidcSageBionetworksAwsInfra` above, except for the repo in the `Repositories` param.   Maybe we can just create one 
 `GithubOidcSharedCloudformationS3Bucket` resource and pass multiple repos to the  `Repositories` parameter?  it would be something like this..

```
  TemplatingContext:
    GitHubOrg: "Sage-Bionetworks"
    Repositories:
      - name: "aws-infra"
        branch: "master"
      - name: "service-catalog-library"
        branch: "master"
  DefaultOrganizationBinding:
    Account: !Ref AdminCentralAccount
    Region: us-east-1
```
I'm not sure that would work because I haven't tested it yet.  I'll test and let you know.

Reply:
That'd be nice! Would we not have to relax the policy a bit ('arn:aws:s3:::bootstrap-awss3cloudformationbucket-19qromfd235z9/service-catalog-library/*' -> 'arn:aws:s3:::bootstrap-awss3cloudformationbucket-19qromfd235z9/*' (at the risk of overwriting another repos artifacts)?

Reply:
I ran a few tests.  Currently the `github-oidc-provider-access.njk` file does not support multiple repositories.  I attempted to change the nunjucks template to add support for multiple repos however I was not able to get it to work. i can't tell if it's something with nunjucks itself or the integration between org formation and nunjucks but something is not working.  I've create issue IT-2768 to try another time.  For now let's just move forward with one of these configs per repo.

Reply:
OK. Thanks!